manifests/g/Googel/OneVPN/1.3.36.292/Googel.OneVPN.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Googel.OneVPN

```

```suggestion
PackageIdentifier: Google.OneVPN
```
Please update file paths to match this change.